usefulUtilities/sitV2/appResources/appData/resources/sittableUIClient.js

```diff
@@ -92,7 +92,7 @@
             if (DEBUG) {
                 console.log("MOUSE RELEASE");
             }
-            if (event.isPrimaryButton) {
+            if (event.isPrimaryButton && Vec3.distance(MyAvatar.position,Entities.getEntityProperties(_this.entityID,["position"]).position) <= 5) {
```

Comment:
```suggestion
            if (event.isPrimaryButton &&
                Vec3.distance(MyAvatar.position, Entities.getEntityProperties(_this.entityID,["position"]).position) <= MAX_CLICK_DISTANCE_M) {
```

and then add a `MAX_CLICK_DISTANCE_M` var at the top of the file, perhaps right below line 20.

Reply:
Ack